c7n/resources/accessanalyzer.py

```diff
@@ -0,0 +1,58 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+import logging
+from c7n.manager import resources
+from c7n import query
+from c7n.utils import local_session
+
+log = logging.getLogger('custodian.access-analyzer')
+
+class DescribeAccessanalyzerFinding(query.DescribeSource):
+
+    def resources(self, query):
+        analyzer_arn = self.get_analyzer_arn()
+        if not analyzer_arn:
+            return ()
+        if not query:
+            query = {}
+        query['analyzerArn'] = analyzer_arn
+        return super().resources(query)
+
+    def get_analyzer_arn(self):
+        """ Find Active Access Analyzer ARN
+        """
+        client = local_session(self.manager.session_factory).client('accessanalyzer')
+        analyzers = client.list_analyzers().get('analyzers', ())
+        found = False
+        for analyzer in analyzers:
+            if analyzer['status'] != 'ACTIVE':
+                continue
+            # If this account is the Management/delegated administrator for IAM Access Analyzer,
+            #  organization analyzer is prefered
+            if analyzer['type'] == 'ORGANIZATION':
+                found = analyzer
+                break
+            found = analyzer
+        if not found:
+            log.warning("policy: {} no active access analyzer found in account".format(
+                self.manager.ctx.policy.name))
+            return None
+        return found['arn']
+
+
+@resources.register("iam-access-analyzer-finding")
```

Comment:
thoughts on shortening the resource name by dropping the `iam-` prefix

Reply:
Yeah I agree, I thought we needed to name it by its full name
will rename it now.